### PR TITLE
Update Teams-add-in-for-Outlook.md

### PR DESCRIPTION
**Pull request by carolynblanding**
Adding workaround to add agent string to EWS Allow list per Kathy Zhang (kaiz@microsoft.com) until development work https://skype.visualstudio.com/SCC/_workitems/edit/1876357 can be completed to reduce case escalations. 

@LolaJacobsen @kenwith

### CODE REVIEW

**Comment by carolynblanding** [comment]
Per Kathy Zhang correct syntax should for EWS Rule should be *SchedulingService*. 

Updated with new commit.

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/Teams-add-in-for-Outlook.md [hunk 1]
```diff
@@ -166,4 +166,6 @@ If the add-in still does not show, use the following steps to verify the registr
 
 ### Delegate scheduling does not work
 
-If your administrator has configured Microsoft Exchange to [control access to Exchange Web Server (EWS)](https://docs.microsoft.com/exchange/client-developer/exchange-web-services/how-to-control-access-to-ews-in-exchange), a delegate won't be able to schedule a Teams meeting on behalf of the boss. The solution for this configuration is under development and will be released in the future. 
+If your administrator has configured Microsoft Exchange to [control access to Exchange Web Server (EWS)](https://docs.microsoft.com/exchange/client-developer/exchange-web-services/how-to-control-access-to-ews-in-exchange), a delegate won't be able to schedule a Teams meeting on behalf of the boss. The solution for this configuration is under development and will be released in the future. As a workaround, your administrator can add the following string to the EWS Allow List "*SchedulingService*".
```
Comment (JohanFreelancer9) [inline Teams/Teams-add-in-for-Outlook.md:169]:
```suggestion
If your administrator has configured Microsoft Exchange to [control access to Exchange Web Server (EWS)](https://docs.microsoft.com/exchange/client-developer/exchange-web-services/how-to-control-access-to-ews-in-exchange), a delegate won't be able to schedule a Teams meeting on behalf of the boss. The solution for this configuration is under development and will be released in the future. As a workaround, your administrator can add the following string to the EWS Allow List: "*SchedulingService*". 
```

**Comment by scanum** [comment]
@LanaChin PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**